store/sqlstore/post_store.go

```diff
@@ -958,57 +958,55 @@ func (s *SqlPostStore) Search(teamId string, userId string, params *model.Search
 	})
 }
 
-func (s *SqlPostStore) AnalyticsUserCountsWithPostsByDay(teamId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query :=
-			`SELECT DISTINCT
+func (s *SqlPostStore) AnalyticsUserCountsWithPostsByDay(teamId string) (model.AnalyticsRows, *model.AppError) {
+	query :=
+		`SELECT DISTINCT
 			        DATE(FROM_UNIXTIME(Posts.CreateAt / 1000)) AS Name,
```

Comment:
You should be maintaining the query identation consistent, here and in the rest of the changes of this function.